src/persist/src/indexed/snapshot.rs

```diff
@@ -196,57 +163,16 @@ impl Snapshot<Vec<u8>, Vec<u8>> for TraceSnapshot {
 // This intentionally stores the batches as a VecDeque so we can return the data
 // in roughly increasing timestamp order, but it's unclear if this is in any way
 // important.
+#[derive(Debug)]
```

Comment:
nit I think we can take the comment above here and above UnsealedSnapshotIter and move it to be above BatchesIterBatch

Reply:
oh yup, missed this. will do